.env

```
@@ -0,0 +1,8 @@
+#DB_HOST=10.250.0.103
+#DB_NAME=easyevent
+#DB_USER=easyevent
+#DB_PASS=a[ez-4.wBhai48M8
+ DB_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)